backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -47,60 +29,22 @@
 /** Class representing the notification system capable of sending mail to the user. */
 public class EmailNotifier {
 
-  private static final String APPLICATION_NAME = "Ad-lib";
+  //   TODO(#35): Create a dummy email account for ad lib itself to send emails.
   private static final String APPLICATION_EMAIL = "Adlib-Step@gmail.com";
```

Comment:
Change this to your email? I feel like a random email address shouldn't be in our public repo.

Reply:
Okay, I've changed adlib-Step@gmail.com to my ldap